packages/main/src/plugin/telemetry/telemetry.ts

```
@@ -450,16 +452,48 @@ export class Telemetry {
       os_distribution,
       locale,
+      using_custom_certificates,
     };
   }
 
+  protected async hasCustomCertificates(): Promise<boolean> {
+    const certDirs = [path.join(os.homedir(), '.config', 'containers', 'certs.d'), '/etc/containers/certs.d'];
+    const certExt = /\.(?:crt|cert|key)$/i;
+
+    for (const dir of certDirs) {
+      let entries;
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)